puffin_egui/src/lib.rs

```diff
@@ -102,6 +100,7 @@ use egui::*;
 use puffin::*;
 use std::{
     collections::{BTreeMap, BTreeSet},
+    fmt::Write,
```

Comment:
While we're at it, this is a nicer way to import traits imho, as it makes it clear that it is a trait

```suggestion
    fmt::Write as _,
```

Reply:
Ok, I wasn't aware of this difference.